src/components/tabs.md

```diff
@@ -172,11 +217,21 @@ Reference implementations are intended to demonstrate **what needs to be achieve
 
 ## Related research
 
-This topic does not yet have any related research available.
+During the development of the tabs implementation, the BBC accessibility team undertook research and hosted two days of usability testing with a number of blind and partially sighted participants.
+
+These participants were presented with four variations of the tab interface. Version 1 appeared as a tab interface but retained the semantics and behavior of an interactive table of contents (see [Initial markup](#initial-markup)). Version 2 implemented all of the recommended semantics and behaviors of the [ARIA Authoring practices](https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel) guide, including tabs only being focusable/selectable by arrow key. The remaining versions constituted compromise positions: Version 3 implemented the tab semantics but behaved as a table of contents (moving focus to panels on tab activation). Version 4 implemented [ARIA Authoring practices tabs](https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel) except without arrow navigation (tabs remained focusable using the <kbd>Tab</kbd> key).
```

Comment:
These participants were presented with **three** variations of the tab interface. Version 1 implemented all of the recommended semantics and behaviours of the ARIA Authoring practices guide, including tabs only being focusable/selectable by arrow key. Version 2 constituted compromise positions implementing the tab semantics but behaved as a table of contents (moving focus to panels on tab activation). Version 3 appeared as a tab interface but retained the semantics and behaviour of an interactive table of contents (see Initial markup).